go.mod

```diff
@@ -32,8 +32,10 @@ require (
 	github.com/xeipuuv/gojsonschema v0.0.0-20180618132009-1d523034197f // indirect
 	go.opencensus.io v0.22.0
 	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
-	golang.org/x/sys v0.0.0-20190507160741-ecd444e8653b
+	golang.org/x/sys v0.0.0-20190916202348-b4ddaad3f8a3
 	google.golang.org/grpc v1.20.1
 	gotest.tools v2.2.0+incompatible // indirect
 	k8s.io/kubernetes v1.13.0
 )
+
+replace github.com/Microsoft/go-winio => github.com/kevpar/go-winio v0.4.12-0.20190918232139-d912bb6347a9
```

Comment:
?

Reply:
I needed this to pull in my go-winio changes before the PR is merged. It will be removed once that PR is in.

Reply:
Still learning how to do this kind of thing well with modules!

Reply:
Oh that makes sense.